docs/stats/index.rst

```diff
@@ -77,8 +80,42 @@ with the Li \& Ma formula:
    >>> significance_on_off(n_on=18, n_off=97, alpha=1. / 10, method='lima')
    2.2421704424844875
 
+Confidence Intervals
+--------------------
+
+Assume you measured 6 counts in a Poissonian counting experiment with an
+expected background :math:`b = 3`. Here's how you compute the 90% upper limit
+on the signal strength :math:`\\mu`:
```

Comment:
Is it possible to add a "FC for dummies" functions for the common cases?
Something like this:
```python
excess_lo, excess_hi = fc_excess_limits_poisson(n_on, mu_background, cl)
excess_lo, excess_hi = fc_excess_limits_poisson_on_off(n_on, n_off, alpha, cl)
```
I guess it's still a few lines to implement based on what's here now, and it's non-trivial to implement because some extra parameters like acceptance interval range and spacing has to be chosen so that it's reasonable efficient and accurate?

If you don't have time any more to add this, just let me know, and I'll try to code it up tonight.

Concerning the functions that compute acceptance intervals, it could be useful to add a sentence like:
"Acceptance intervals are used in the Feldman-Cousins computation of confidence intervals. Usually the confidence intervals are the result of interest."